src/hardware/link-mso19/api.c

```diff
@@ -92,6 +93,24 @@ static const int32_t trigger_matches[] = {
 	SR_TRIGGER_ONE,
 };
 
+static const char *logic_thresholds[] = {
+	"1.2V Logic",
+	"1.5V Logic",
+	"1.8V Logic",
+	"2.5V Logic",
+	"3.0V Logic",
+	"3.3V/5.0V Logic",
+};
+
+static const uint16_t logic_threshold_values[] = {
+	0x600,
```

Comment:
is there a way to write this that exposes whatever unit this is in ? not sure if  `600*256/100`  would integer promotion /cast issues, but maybe at least a comment that mentions the scaling

Reply:
I added a comment for it. These values come from USB captures and could be considered to have some constant times mV value, but it seems that would complicate things even more to try and explain it. So I think a comment will work fine.